kola/tests/misc/tls.go

```diff
@@ -35,7 +35,7 @@ func init() {
 		ClusterSize:    1,
 		Name:           "coreos.tls.fetch-urls",
 		Flags:          []register.Flag{register.RequiresInternetAccess}, // Networking outside cluster required
-		ExcludeDistros: []string{"rhcos"},                                // wget not included in RHCOS
+		ExcludeDistros: []string{"rhcos", "fcos"},                        // wget not included in *COS
```

Comment:
@arithx I feel that at this point this would be better split in a curl test and in a wget one, WDYT?

Reply:
@lucab yeah, we should probably split that in the future. I'll open an issue for updating / creating tests on FCOS.

Reply:
Similarly for the NFS one, I think we should maybe rename that one to e.g. "nfs-server-enablement" and have another simpler `systemd.enable-service` that targets a different service (or maybe even just a dummy service that we drop in via Ignition).